src/views/v2/list.php

```diff
@@ -12,25 +12,55 @@
  * @version TBD
  *
  */
+use Tribe\Events\Views\V2\Rest_Endpoint;
 
+/**
+ * @todo  remove this and properly enqueue assets
+ */
+tribe_asset_enqueue( 'tribe-events-views-v2-manager' );
+
+$events_label_plural = tribe_get_event_label_plural();
 $events = $this->get( 'events' );
 
-$this->template( 'events-bar' );
+?>
+<form
```

Comment:
I don't think semantically wrapping the entire list view in a `<form>` is going to work. Is there a specific reason why we're doing this?

Reply:
I wanted customers to be able to include a field in any place inside of the view manager container, which the whole view been a form allows us to handle both with JS active and inactive since when the JS is deactivated it would just be a normal request to the backend.

Reply:
I don't see a submit button anywhere in the list view markup, the navigation is done by links. But also I'd like to chat with you about this and the need for allowing customers to include a field anywhere.

Reply:
Lets do that on monday? There are some good reasons for this, it might seem weird I know.

Reply:
Agree with Paul on this, this is really funky and not conforming to best practices.

Reply:
Added a `@todo` there and we can chat afterward about this one.